Modules/TopGear/TopGear.lua

```diff
@@ -183,20 +183,31 @@ function CraftSim.TOPGEAR:GetProfessionGearFromInventory(recipeData)
 
         for bag = BANK_CONTAINER, NUM_BAG_SLOTS + NUM_BANKBAGSLOTS do
             for slot = 1, C_Container.GetContainerNumSlots(bag) do
-                local itemLink = C_Container.GetContainerItemLink(bag, slot)

```

Comment:
I am currently asking around in the ui dev discord if this function is always returning a link
or if is also dependent on the cached item info one has to trigger the load of
https://warcraft.wiki.gg/wiki/API_C_Container.GetContainerItemLink

Reply:
I didn't really understand that, but like that it works better :D